<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CTC Testbench</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="MainColumnsLayout" stretch="6,0,1">
      <item>
       <layout class="QVBoxLayout" name="TB_inputsLayout" stretch="0,1,2">
        <item>
         <layout class="QHBoxLayout" name="OccupiedBlocksLayout">
          <item>
           <widget class="QLabel" name="OccupiedBlocksInputLabel">
            <property name="text">
             <string>Occupied Blocks:</string>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="OccupiedBlocksInput"/>
          </item>
          <item>
           <widget class="QPushButton" name="UpdateOccupiedBlocksButton">
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="OccupiedBlocksInstructionLabel">
          <property name="text">
           <string>(Enter the blocks that are currently occupied, seperated by commas)</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="TicketSaleLayout" stretch="1,2,2,0">
          <item>
           <widget class="QLabel" name="TicketSalesLabel">
            <property name="text">
             <string>Ticket Sales:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="LineSelect">
            <item>
             <property name="text">
              <string>Select Line</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="TicketSalesField"/>
          </item>
          <item>
           <widget class="QPushButton" name="UpdateTicketSalesButton">
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="TB_OutputsLayout">
        <item>
         <widget class="QLabel" name="OuptutsLabel">
          <property name="text">
           <string>Outputs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="AuthorityLabel">
          <property name="text">
           <string>Authority</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="SpeedLabel">
          <property name="text">
           <string>Suggested Speed</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
